LEGALNOTICE.md

```diff
@@ -0,0 +1,69 @@
+COPYRIGHT
+---------
+
+OWASP Zed Attack Proxy (ZAP)
+
+The software package is:
+
+    Copyright (C) 2010  Simon Bennetts
```

Comment:
Oops. I forgot to include a mention of Paros and the Clarified Artistic license. Any thoughts on the wording?